web_programming/emails_from_url.py

```diff
@@ -1,103 +1,96 @@
-"""Get the site emails from URL."""
-__author__ = "Muhammad Umer Farooq"
-__license__ = "MIT"
-__version__ = "1.0.0"
-__maintainer__ = "Muhammad Umer Farooq"
-__email__ = "contact@muhammadumerfarooq.me"
-__status__ = "Alpha"
+"""
+Author - AyushSehrawat
```

Comment:
Perhaps instead of deleting the work of someone else, you should create a `emails_from_url_alt.py` implementation.  I like the current implementation and it would be good to be able to see the two side-by-side.

Reply:
So you mean, close this pr and create a new one?

Reply:
@cclauss So close this pr ? Or make a new file and commit using this pr , and let the original be same ?